src/lightning/pytorch/utilities/combined_loader.py

```diff
@@ -154,30 +154,30 @@ class _CombinationMode(TypedDict):
     iterator: Type[_ModeIterator]
 
 
-_supported_modes = {
+_SUPPORTED_MODES = {
     "min_size": _CombinationMode(fn=min, iterator=_MinSize),
     "max_size_cycle": _CombinationMode(fn=max, iterator=_MaxSizeCycle),
     "max_size": _CombinationMode(fn=max, iterator=_MaxSize),
     "sequential": _CombinationMode(fn=sum, iterator=_Sequential),
 }
-
 _LITERAL_SUPPORTED_MODES = Literal["min_size", "max_size_cycle", "max_size", "sequential"]
 
 
 class CombinedLoader(Iterable):
-    """Combines different iterables under custom sampling modes.
+    """Combines different iterables under specific sampling modes.
+
+    The following modes are supported:
+        * ``min_size``: stops after the shortest iterable (the one with the lowest number of items) is done.
```

Comment:
The rendering for this is a bit funky. I couldn't find a way to fix it. Suggestions welcome. Looks like a problem in our theme

Reply:
Would you like to have the modes listed here or below under the mode arg? 

This

<img width="805" alt="image" src="https://user-images.githubusercontent.com/5495193/224769669-310ef72b-85a6-4c9a-b1f8-cbc0b96fe0be.png">


vs.

<img width="712" alt="image" src="https://user-images.githubusercontent.com/5495193/224769541-b8c98d52-ebac-4246-a70e-7c9c461ecae6.png">

Reply:
I like it better above, but I don't mind either option.

Reply:
We'll need to skip the line-too-long rule